frontend/src/views/SettingsView.vue

```diff
@@ -1,7 +1,110 @@
 <template>
 	<div class="page_box_wrapper">
 		<div class="page_box">
-			<h1>This is a profile settings page</h1>
+			<h1>SETTINGS</h1>
+			<ProfileSettingAvatar :profile_picture="me.avatar" />
+			<div class="c_block c_form_group tac">
+				<ChangeAvatar />
+			</div>
+			<form
+				method="Post"
+				action=""
+				class="c_block c_form_group"
+				@submit.prevent="updateProfile(me.id, updateProfileForm)"
+			>
+				<InputField
+					v-if="me.is_intra"
+					label="intra name"
+					v-model="me.intra_login"
+					:is_disabled="true"
+				/>
+				<InputField
+					label="display_name"
+					:modelValue="me.name"
+					v-model="updateProfileForm.name"
```

Comment:
vscode complains about this:
> An object literal cannot have multiple properties with the same name

i think it's about using :modelValue and v-model on the same component.. but not sure..

Reply:
if you fix your ESlint it should show it in clion as well :P

Reply:
what about this?
:modelValue="me.name"
@update:modelValue="updateProfileForm.name = $event"

Reply:
it does not give an error 👍 
haven't ran the container, but assuming you actually filled in something and pressed save to verify that it works, it looks good to me :)